services/node/node-current.tester.js

```diff
@@ -13,98 +13,164 @@ function expectSemverRange(message) {
+      tag: '',
+      registry: '',
+    })
+  )
   .intercept(mockCurrentSha(13))
   .expectBadge({ label: 'node', message: `12`, color: `yellow` })
+
+t.create("gets the tagged release's node version version of ionic - live")
+  .get('/ionic/testing.json')
+  .expectBadge({ label: 'node@testing' })
   .afterJSON(json => {
     expectSemverRange(json.message)
   })
 
 t.create('engines satisfies current node version - tagged')
-  .get('/ionic/next.json')
-  .intercept(mockPackageData(`ionic`, `>=0.4.0`, undefined, `next`))
+  .get('/ionic/testing.json')
+  .intercept(
+    mockPackageData({
+      packageName: 'ionic',
+      engines: '>=0.4.0',
+      tag: 'testing',
+    })
+  )
   .intercept(mockCurrentSha(13))
-  .expectBadge({ label: 'node@next', message: `>=0.4.0`, color: `brightgreen` })
-  .afterJSON(json => {
-    expectSemverRange(json.message)
+  .expectBadge({
+    label: 'node@testing',
+    message: `>=0.4.0`,
+    color: `brightgreen`,
   })
 
 t.create('engines not satisfies current node version - tagged')
-  .get('/ionic/next.json')
-  .intercept(mockPackageData(`ionic`, `12`, undefined, `next`))
+  .get('/ionic/testing.json')
+  .intercept(
+    mockPackageData({
+      packageName: 'ionic',
+      engines: '12',
+      tag: 'testing',
+    })
+  )
   .intercept(mockCurrentSha(13))
-  .expectBadge({ label: 'node@next', message: `12`, color: `yellow` })
+  .expectBadge({ label: 'node@testing', message: `12`, color: `yellow` })
+
+t.create("gets the tagged release's node version of @cycle/core - live")
+  .get('/@cycle/core/canary.json')
+  .expectBadge({ label: 'node@canary' })
   .afterJSON(json => {
     expectSemverRange(json.message)
   })
 
 t.create('engines satisfies current node version - scoped and tagged')
   .get('/@cycle/core/canary.json')
-  .intercept(mockPackageData(`core`, `>=0.4.0`, `@cycle`, `canary`))
+  .intercept(
+    mockPackageData({
+      packageName: 'core',
+      engines: '>=0.4.0',
+      scope: '@cycle',
+      tag: 'canary',
+    })
+  )
   .intercept(mockCurrentSha(13))
   .expectBadge({
     label: 'node@canary',
     message: `>=0.4.0`,
     color: `brightgreen`,
   })
-  .afterJSON(json => {
-    expectSemverRange(json.message)
-  })
 
 t.create('engines not satisfies current node version - scoped and tagged')
   .get('/@cycle/core/canary.json')
-  .intercept(mockPackageData(`core`, `12`, `@cycle`, `canary`))
+  .intercept(
+    mockPackageData({
+      packageName: 'core',
+      engines: '12',
+      scope: '@cycle',
+      tag: 'canary',
+    })
+  )
   .intercept(mockCurrentSha(13))
   .expectBadge({ label: 'node@canary', message: `12`, color: `yellow` })
+
+t.create('gets the node version of passport from a custom registry - live')
+  .get('/passport.json?registry_uri=https://registry.npmjs.com')
+  .expectBadge({ label: 'node' })
   .afterJSON(json => {
     expectSemverRange(json.message)
   })
 
 t.create('engines satisfies current node version with custom registry')
```

Comment:
We really only need the mocked tests to validate logic within `node-version-color`, and we don't want to have any extra mocked tests that are duplicating scenarios which are explicitly validated by a live test.

As such let's remove the test at the bottom of the file that's mocking an invalid package, and I'm also wondering if this one can be deleted too given the live test above

Reply:
I agree will remove them